css/index.css-copy.css

```diff
@@ -0,0 +1,433 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+v2.0 | 20110126
+License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+ margin: 0;
+ padding: 0;
+ border: 0;
+ font-size: 100%;
+ font: inherit;
+ vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+ display: block;
+}
+body {
+ line-height: 1;
+}
+ol, ul {
+ list-style: none;
+}
+blockquote, q {
+ quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+ content: '';
+ content: none;
+}
+table {
+ border-collapse: collapse;
+ border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+ box-sizing: border-box;
+}
+
+html, body {
+ height: 100%;
+ font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+ font-family: 'Bangers', cursive;
+ letter-spacing: 1px;
+ margin-bottom: 15px;
+}
+
+/*----------------------------------------------------------------------------*/
+
+.container {
+    margin: auto;
+    width: 60%;
+    font-family: 'Exo 2', sans-serif;
+}
+
+/* At 800px And below do the following: Tablet */
+@media (max-width: 800px) {
+    .container { 
+        width: 800px;
+    }
+}
+
+/* At 500px and below do the following: Phone  */
+@media (max-width: 500px) {
+    .container {
+        width: 500px
+    }
+}
+
+header {
+    box-sizing: border-box;
+    border-bottom: 2px solid black;
+    padding-bottom: 70px;
+    text-align: center;
+
+}
+
+/* At 800px and below do the following: Tablet */
+@media (max-width: 800px) {
+    header {
+        width: 700px;
```

Comment:
This is a hard coded width, i would replace this with a `max-width:` or with `700%` instead of `700PX`